.prettierrc

```diff
@@ -0,0 +1,6 @@
+{ 
+  "trailingComma": "es5",
+  "tabWidth": 2,
```

Comment:
isn't `2` the default?

Reply:
I just wanted it to be explicit, I will remove it.